rust/src/logger.rs

```diff
@@ -0,0 +1,152 @@
+//! This module implements logging facilities.
+//!
+//! # FFI
+//!
+//! [`stracciatella::c::logger`] contains a C interface for this module.
+//!
+//! [`stracciatella::c::logger`]: ../c/logger/index.html
+
+use lazy_static::lazy_static;
+use log::*;
+use simplelog::*;
```

Comment:
It's better to avoid `*` imports unless it's something like a prelude. (`c::common` is like a prelude)
It's hard to see which crate log stuff comes from like this.
There is also potential for conflict when a new symbol is added to one of those crates.